tests/conftest.py

```diff
@@ -38,14 +38,26 @@ def local_server(httpserver) -> LocalHttpServer:
 
 
 @pytest_asyncio.fixture
-async def websocket():
-    """Return a websocket connection to the browser on localhost."""
+async def websocket_connection():
+    """ Return a websocket connection to the browser on localhost without an
+    active BiDi session.
+    """
     port = os.getenv("PORT", 8080)
     url = f"ws://localhost:{port}"
     async with websockets.connect(url) as connection:
         yield connection
 
 
+@pytest_asyncio.fixture
+async def websocket(websocket_connection):
+    """Return a websocket with an active BiDi session."""
+    await execute_command(websocket_connection, {
+        "method": "session.new",
+        "params": {}
+    })
+    yield websocket_connection
```

Comment:
return

`yield` is only needed if you have more statements afterwards

Reply:
fixed